### ISSUE TITLE
How is the official merging now?

### ISSUE THREAD

**Issue by siriuscn** [iains/gcc-darwin-arm64#104]
Thanks for your arm64 branch. I built the cross compiler chains for PowerPC Darwin on Apple Silicon host with your great branch.

But I am puzzled how is the official merging now. GCC 13 official version is going to release now. Did they reject this branch?

**Comment by iains** [comment]
No, it has not been rejected: The branch has not been submitted to merge (there is still work to do to complete it, and this is only "spare time" at present)

**Comment by siriuscn** [comment]
What can i do for the job?

**Comment by ilg-ul** [comment]
That would be really nice, Homebrew GCC and indirectly xPack GCC would greatly benefit from simplified releases.

If there is anything I can do to help, please let me know.

**Comment by cooljeanius** [comment]
> May I help you? What can i do for the merge?
> 
> GCC 13.1 was released today.

> That would be really nice, Homebrew GCC and indirectly xPack GCC would greatly benefit from simplified releases.
> 
> If there is anything I can do to help, please let me know.

Well, do either of you have experience with the process of submitting patches upstream to GCC? Mostly it's just the whole process of (formatting a patch according to patch submission guidelines -> testing it -> sending it to the gcc-patches mailing list -> getting it approved -> confirming that you have a valid copyright assignment and/or DCO -> actually committing it) that takes awhile...

**Comment by iains** [comment]
> > May I help you? What can i do for the merge?
> > GCC 13.1 was released today.
> 
> > That would be really nice, Homebrew GCC and indirectly xPack GCC would greatly benefit from simplified releases.
> > If there is anything I can do to help, please let me know.
> 
> Well, do either of you have experience with the process of submitting patches upstream to GCC? Mostly it's just the whole process of (formatting a patch according to patch submission guidelines -> testing it -> sending it to the gcc-patches mailing list -> getting it approved -> confirming that you have a valid copyright assignment and/or DCO -> actually committing it) that takes awhile...

It is not as simple as a "patch to fix a bug " (which is covered by the procedures above) -  this is a whole new port and needs multiple co-operating patches (although we are [and have already posted some] working on splitting out parts that can be posted independently)
(with much help from @fxcoudert who is already familiar with the port).

- I really appreciate the offer of help, but - as this stage - unfortunately it is a case of me continuing to clean up the code and split out pieces to post.  $dayjob willing, hopefully we will post before end of stage #1 gcc14.

**Comment by ugomancz** [comment]
Hi guys, it's been two years, have things progressed regarding the project status and the merge to upstream? Is there any ETA (even a wild guess) for when this will make it to upstream?

**Comment by iains** [comment]
> Hi guys, it's been two years, have things progressed regarding the project status and the merge to upstream? Is there any ETA (even a wild guess) for when this will make it to upstream?

I'm trying to do some of the smaller dependent pieces in GCC-16, but there is no realistic forecast at the moment - this is not a small piece of work to upstream and most of my $volunteer time seems to get soaked up trying to keep up with SDK/XCode changes etc.

**Comment by Jason2866** [comment]
Thank you for all the efforts putting into this!
